src/lib/y2storage/dialogs/guided_setup/select_scheme.rb

```diff
@@ -0,0 +1,200 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact Novell, Inc.
+#
+# To contact Novell about this file by physical or electronic mail, you may
+# find current contact information at www.novell.com.
+
+require "yast"
+require "y2storage"
+require "y2storage/dialogs/guided_setup/base"
+
+Yast.import "InstExtensionImage"
+
+module Y2Storage
+  module Dialogs
+    class GuidedSetup
+      # Dialog to select partitioning scheme.
+      class SelectScheme < Base
+        def encryption_handler
+          widget_update(:password, widget_value(:encryption), attr: :Enabled)
+          widget_update(:repeat_password, widget_value(:encryption), attr: :Enabled)
+        end
+
+        def back_handler
+          unload_cracklib
```

Comment:
Wouldn't it be better to redefine `#finish_dialog` or `#close_dialog` in order to unload cracklib? It sounds more robust to me than doing it in every possible handler.